styles.css

```diff
@@ -102,5 +118,242 @@ body {
   height: 40px;
 }
 
+.my-works-section {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  background: #fffefe;
+}
+
+.recent-works {
+  margin: 100px 20px 10px 20px;
+  grid-column: span 2;
+}
+
+.recent-works h1{
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px; 
+  text-align: center;
+  margin-bottom: 50px;
+  padding: 30px;
+  border-bottom: 1px solid #212121;
+}
+
+.recent-works img {
+  width: 100%;
+}
+
+.recent-works h3 {
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091E42;
+  text-align: start;
+}
+
+.recent-works p{
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091E42;
+  text-align: start;
+  margin: 10px 0;
+}
+
+.recent-works button {
+  background: #F55800;
+  color: #fff;
+  padding: 12px;
+  margin: 10px 0;
+  border: none;
+  font-family: 'Inter' , sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  
+}
+
+.recent-works a {
+  color: #fff;
+}
```

Comment:
- [ ] Your button should follow the button styling: Active, Hover and Pressed as provided from Figma design. This will make your webpage more interactive and accessible.

Reply:
how do i go about achieving this, i cannot find where they provided the styling for all of these attributes you listed, can you please give me a pointer to achieving this or a way to do it. @ishimwezachee

Reply:
the reason i asked the earlier question is because there are no visible different styling for when an hover is made which should indicated hover or active as all of the styling are the same. so basically the styling for the active, hover and enabled are the same styling for the button itself